services/analyzer/mysql-table-constraints-getter.js

```diff
@@ -0,0 +1,54 @@
+function MysqlTableConstraintsGetter(databaseConnection) {
+  const queryInterface = databaseConnection.getQueryInterface();
+
+  this.convertToUniqueIndexArray = (constraints) => {
```

Comment:
[Readability] may you add a little notice there to help dev going through ? (Describing the type of data computed there for example)